integration/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "k8s-gateway-api-tests"
+version = "0.4.0"
+edition = "2021"
+license = "Apache-2.0"
+publish = false
+
+[dependencies]
+k8s-gateway-api = { path = "../k8s-gateway-api" }
+kube = { version = "0.73", default-features = false, features = ["derive"] }
+k8s-openapi = { version = "0.15", features = ["schemars"] }
+
```

Comment:
Does this crate actually have library code or is it all tests? If the latter, we should be able to move these to dev dependencies.